sources/packages/api/src-sql/Reports/Create-initial-reports-config.sql

```diff
@@ -0,0 +1,15 @@
+--Insert config for disbursement forecast report.
+INSERT INTO
+    sims.reports_config (report_name, report_sql)
+VALUES
+    (
+        'Disbursement_Forecast_Report',
+        'select ds.disbursement_date as "Disbursement Date", dv.value_code as "Funding Type", sum(dv.value_amount) as "Amount", count(ap.id) as "Count" from sims.disbursement_schedules ds 
```

Comment:
Can we have the SQL queries inserted as they would be formatted by our VS code plugin? At least the ones created by the DEV team will easily follow the same syntax.

Reply:
Hope that below image helps 😉 
![image](https://user-images.githubusercontent.com/61259237/170130859-f55e7ccc-57f3-4e16-94f3-dc0ddaeb29fb.png)

Reply:
Ah you meant to say the dynamic-sql to be formatted by vs code and copied in here to the migration sql? right